pkg/controller/reconcile.go

```diff
@@ -0,0 +1,99 @@
+package controller
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"os"
+
+	"github.com/sirupsen/logrus"
+	"github.com/water-hole/ansible-operator/pkg/events"
+	"github.com/water-hole/ansible-operator/pkg/proxy/kubeconfig"
+	"github.com/water-hole/ansible-operator/pkg/runner"
+	"github.com/water-hole/ansible-operator/pkg/runner/eventapi"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+// ReconcileAnsibleOperator - object to reconcile runner requests
+type ReconcileAnsibleOperator struct {
```

Comment:
Super minor nit, but I think this should be a noun rather than a verb, I read this as a function. Maybe something like `AnsibleOperatorReconciler`?